code/modules/vehicles/mecha/equipment/tools/other_tools.dm

```diff
@@ -156,13 +156,15 @@
 	//how much the armor of the mech is modified by
 	var/datum/armor/armor_mod
 
+/* MOVED TO ZZZSPLURT MODULAR FILE SYSTEM
```

Comment:
Considering you just added `chassis.update_move_speed()` to both of these procs, you don't have to comment, them, just overload them with your new calls (see comments below)